src/category_theory/category.lean

```diff
@@ -36,25 +36,29 @@ def_replacer obviously
 class has_hom (obj : Type u) : Type (max u (v+1)) :=
 (hom : obj → obj → Type v)
 
-infixr ` ⟶ `:10 := has_hom.hom -- type as \h
+class category_struct (obj : Type u)
```

Comment:
Is there a reason not to leave the `⟶` notation here and use it in the definition of `category_struct`?

Reply:
No reasons, I'll fix it.